test/systemtest/earlgrey/test_sim_verilator.py

```diff
@@ -144,11 +145,14 @@ def find_in_uart0(self,
             # The default filter function for all device software in OpenTitan.
             filter_func = utils.filter_remove_device_sw_log_prefix
 
-        return utils.find_in_files([self._uart0_log],
-                                   pattern,
-                                   timeout,
-                                   filter_func=filter_func,
-                                   from_start=from_start)
+        try:
+            return utils.find_in_files([self._uart0_log],
+                                    pattern,
+                                    timeout,
+                                    filter_func=filter_func,
+                                    from_start=from_start)
```

Comment:
Nit: Lines 150-153 need indenting to match.

Also, sorry for not spotting this call site!